### ISSUE TITLE
The `trace` Module - stateDiff

### ISSUE THREAD

**Issue by adirmola** [paritytech/parity#8495]
> - **Which Parity version?**: 1.8.11
> - **Which operating system?**: Windows
> - **How installed?**: via installer
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: ethereum
> - **Did you try to restart the node?**: no

Hi, I'm trying to explore the influence of transaction on accounts state.

Here is how Etherscan represent the transaction:
https://etherscan.io/tx/0xa567744e4a4b267fecbfd95e6df4b058adb9afa0f22355d11cfdc2bceed446b5

I had running the following query:
```
payload =  '{"method":"trace_replayTransaction","params":["0xa567744e4a4b267fecbfd95e6df4b058adb9afa0f22355d11cfdc2bceed446b5",["stateDiff"]],"id":1,"jsonrpc":"2.0"}'
headers = {"Content-Type": "application/json"}
res = requests.post(url, data=payload, headers=headers)
```
following is the response (I copied only the contract section):
```
"0x423e4322cdda29156b49a17dfbd2acc4b280600d":{
"balance":"=",
"code":"=",
"nonce":"=",
"storage":{
"0xa0b6513ad18878b03f0fa500d8e61c2ffa4661f704d4d2ce327b82c6b99fa9fe":{
"*":{
"from":"0x0000000000000000000000000000000000000000000000000000000000000000",
"to":"0x000000000000000000000000000000000000000000000000000a53becd2d3453"
}
},
"0xaed173b0583e10a71a98e19bb793fb6b30c48f6a00720c598fcae2772c1790b7":{
"*":{
"from":"0x0000000000000000000000000000000000000000000000000000000000000000",
"to":"0x000000000000000000000000000000000000000000000000000014198ea64800"
}
}
}
}
```

The first "from-to" couple should be the change in the token balance of the sender and the second should be of the receiver.
**But, as you can see the, "from" of both is "0" what that wasn't true before execute the transaction.**

**Comment by Tbaut** [comment]
Can you please post the whole response to have a better overview?

**Comment by sorpaas** [comment]
Duplicate of #6520 and should be fixed in #8491. Yay!
